shapely/predicates.py

```diff
@@ -874,11 +876,12 @@ def overlaps(a, b, **kwargs):
     a, b : Geometry or array_like
         Geometry or geometries to check.
     **kwargs
-        See :ref:`NumPy ufunc docs <ufuncs.kwargs>` for other keyword arguments.
+        See :ref:`NumPy ufunc docs <ufuncs.kwargs>` for other keyword
+        arguments.
 
     See Also
     --------
-    prepare : improve performance by preparing ``a`` (the first argument)
+    prepare : improve performance by preparing ``a`` (the 1st argument)
```

Comment:
```suggestion
    prepare : improve performance by preparing ``a`` (the first argument)
```

(going to revert this small change; at the moment we don't use that in shapely, and in general I think the recommendation for small numbers is to use the long version in written text. We can certainly discuss that but let's do in the context of the full docs)